doc/iterations/2024.3/shapes/add_support_for_browse_upload_images_widget.adoc

```diff
@@ -0,0 +1,95 @@
+= Add support for a browse / upload images widget
+
+== Problem
+
+Up until today, there is no specific widget to manage images.
+We use a selection widget that only displays image label and does not show any preview.
+It's not possible to add new images either.
+The only available images are the one programmatically loaded on the backend.
+
+== Key Result
+
+Have a single widget for both selecting an image (with an integrated preview) and contributing new images, either from a web link or from the local disk.
+
+== Solution
+
+Create a new custom widget in Sirius Components.
+
+This new widget will provide the same level of configuration as its EEF equivalent:
+
+* _Label Expression_
+* _Help Expression_
+* _Is Enabled Expression_
+
+* _AddImageOperation_ with a _body_ to specify the action to apply once an image is selected.
+* _RemoveImageOperation_ with a _body_ to specify the action to apply once an image is deleted.
+* _style_ (with its corresponding _conditionalStyles_) to configure the label
+
+This new widget will be based on the UI of the reference widget, using the MUI autocomplete component.
+
+* The selected images will be displayed in _chips_ with preview and label.
```

Comment:
Is the widget proposed here be enough to preview images? Chips seem to only display images in a very small square. I wonder if it will be appropriate.

Reply:
After discussion, no ned to change anything.